config/renovate.json5

```diff
@@ -50,6 +50,26 @@
             "versioningTemplate": "semver-coerced",
             "datasourceTemplate": "github-tags"
         },
+        // Matches github-tags pointing to items in github.com/<org>/<repo>/raw/<tag> (i.e. https://github.com/defenseunicorns/uds-package-postgres-operator/blob/c88f9fd47c2fbc85e86c86c26b2d6948f6220898/zarf.yaml#L17)
+        {
+            "fileMatch": [".*(\\.ya?ml|\\.json)$"],
+            "matchStrings": [
+                // Test: https://regex101.com/r/huQrD4/1
+                "https:\\/\\/github\\.com\\/(?<depName>[^\\/]+\\/[^\\/]+)\\/raw\\/(?<currentValue>[^\\/]+)"
+            ],
+            "versioningTemplate": "semver-coerced",
+            "datasourceTemplate": "github-tags"
+        },
+        // Matches github-tags pointing to items in github.com/<org>/<repo>/releases/download/<tag> (i.e. https://github.com/defenseunicorns/zarf/blob/c8abe31a5c4bbed933d692fcdeeefdc4b31701a5/packages/distros/k3s/zarf.yaml#L17)
+        {
+            "fileMatch": [".*(\\.ya?ml|\\.json)$"],
```

Comment:
In my head im wondering if we want to (this is common anyways) allow this to work for more than just yaml and json and possibly more then just github.com (like gitlab.com or internal instances of SCM for maximum potential for reuse of the common config. 

REgardless thats that out of scope of this. Looks good.